app/assets/javascripts/components/instructor_panel.js.jsx

```diff
@@ -33,6 +33,15 @@ var InstructorPanel = React.createClass({
     };
 
   },
+  mergeQueueItem: function (queue) {
+    var click = function () {
+      if (confirm('Are you sure? This will permanently merge ' + this.props.queueLength
+           + ' request(s) into ' + queue.name + '.')) {
```

Comment:
Maybe add a note here so the instructor knows what the students can expect? "Students will be automatically redirected to the new queue."

Reply:
Done